src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,164 @@
+  <string name="light">Hell</string>
+  <string name="link_copied_to_clipboard">Link in Zwischenablage kopiert</string>
+  <string name="links">Links</string>
+  <string name="merging_FORMAT">Vereine %s</string>
+  <string name="name">Name</string>
+  <string name="network_error_DESC">Netzwerkfehler.</string>
+  <string name="never">Nie</string>
+  <string name="new_updates_available">Neue Updates verfügbar</string>
+  <plurals name="new_updates_DESC_FORMAT">
+    <item quantity="one">%d neues Update.</item>
+    <item quantity="other">%d neue Updates.</item>
+  </plurals>
+  <string name="no_applications_available">Keine Apps verfügbar</string>
+  <string name="no_applications_installed">Keine Apps installiert</string>
+  <string name="no_description_available_DESC">Keine Beschreibung verfügbar.</string>
+  <string name="no_matching_applications_found">Keine passenden Apps gefunden</string>
+  <string name="no_proxy">Kein Proxy</string>
+  <string name="notify_about_updates">Update-Benachrichtigung</string>
+  <string name="notify_about_updates_summary">Bei verfügbaren Updates eine Benachrichtigung anzeigen</string>
+  <string name="number_of_applications">Anzahl der Apps</string>
+  <string name="ok">OK</string>
+  <string name="only_compatible_with_FORMAT">Nur kompatibel mit %s</string>
+  <string name="only_on_wifi">Nur über WLAN</string>
+  <string name="open_DESC_FORMAT">Öffne %s?</string>
+  <string name="other">Andere</string>
+  <string name="parsing_index_error_DESC">Indexdatei konnte nicht analysiert werden.</string>
+  <string name="password">Passwort</string>
+  <string name="password_missing">Passwort fehlt</string>
+  <string name="permissions">Berechtigungen</string>
+  <string name="plus_more_FORMAT">+%d mehr</string>
+  <string name="preferences">Einstellungen</string>
+  <string name="processing_FORMAT">Verarbeite %1$s</string>
+  <string name="project_website">Projekt-Webseite</string>
+  <string name="promotes_non_free_network_services">Bewirbt unfreie Netzwerkservices</string>
+  <string name="promotes_non_free_software">Bewirbt unfreie Software</string>
+  <string name="provided_by_FORMAT">Bereitgestellt von %s</string>
+  <string name="proxy">Proxy</string>
+  <string name="proxy_host">Proxy-Host</string>
+  <string name="proxy_port">Proxy-Port</string>
+  <string name="proxy_type">Proxy-Typ</string>
+  <string name="repositories">Paketquellen</string>
+  <string name="repository">Paketquelle</string>
+  <string name="repository_not_used_DESC">Diese Paketquelle wurde noch nicht verwendet. Bitte Aktivieren um die angebotenen Apps anzusehen.</string>
+  <string name="repository_unsigned_DESC">Unsigniert. App-Liste konnte nicht verifiziert werden. Vorsicht bei Downloads aus unsignierten Paketquellen.</string>
+  <string name="requires_FORMAT">Benötigt %s</string>
+  <string name="save">Speichern</string>
+  <string name="saving_details">Speichere Details</string>
+  <string name="screenshots">Screenshots</string>
```

Comment:
```suggestion
  <string name="screenshots">Bildschirmfotos</string>
```
As per https://en.wiktionary.org/wiki/Bildschirmfoto

Reply:
The english word is more common in computer world than the german translation i think. Not shure what is better here. 🤔 Could probably changed.